KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+---
+kip: 81
+title: Changing GC Voting Method
+author: Yeriel<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+discussion-to: https://govforum.klaytn.foundation/t/discussion-changing-gc-voting-method/19
+status: Last Call
+type: Standards Track 
+category: Core
+created: 2022-09-19
+---
+
+
+## Simple Summary
+Introducing a new governance voting method based on the staking amount and implementation of the governance portal, Klaytn Square 
+ 
+## Abstract
+Klaytn introduces a stake-based governance model that provides voting power to governance participants. Currently, one vote per Governance Council(GC) member was cast. The new method will introduce the voting right that will be exercised based on the staking amount with the maximum cap to prevent an entity from making an arbitrary decision. This model ultimately enables each member to gather support from Klaytn communities in line with their preference. The new system aims to provide responsibility and obligation of voting to Governance Councils. 
```

Comment:
```suggestion
Klaytn introduces a stake-based governance model that provides voting power to governance participants. Currently, one Governance Council (GC) member can cast one vote. The new method will introduce a voting right that will be exercised based on the staking amount with a maximum cap to prevent a certain entity from making an arbitrary decision. This model ultimately enables each member to gather support from Klaytn communities in line with their preference. The new system aims to give responsibility and obligation to the Governance Council members to vote.
```